pkg/operator/controller/controller_cabundle_configmap_test.go

```diff
@@ -28,12 +27,28 @@ func TestDesiredCABundleConfigmap(t *testing.T) {
 		Spec: operatorv1.DNSSpec{},
 	}
 
-	_, cm, err := desiredCABundleConfigMap(dns, true, &sourceConfigmap, destName)
-	if err != nil {
+	desired, cm, err := desiredCABundleConfigMap(dns, true, &sourceConfigmap, destName)
+	if err != nil || desired == false {
 		t.Errorf("Unexpected error : %v", err)
 	} else if diff := cmp.Diff(cm.Data, sourceConfigmap.Data); diff != "" {
 		t.Errorf("Unexpected CA Bundle ConfigMap data;\n%s", diff)
 	} else if diff := cmp.Diff(cm.OwnerReferences, []metav1.OwnerReference{dnsOwnerRef(dns)}); diff != "" {
 		t.Errorf("Unexpected CA Bundle ConfigMap OwnerReference;\n%s", diff)
 	}
+
+	desired, cm, err = desiredCABundleConfigMap(dns, false, &sourceConfigmap, destName)
+	if desired != false || cm != nil || err != nil {
+		t.Errorf("Unexpected error : %v", err)
```

Comment:
Either a comment or a more informative error message prefix would be helpful, here and in the next one.

nit: first word of error messages should be lower-case.

Reply:
Fixed this locally as well; I've downcased the first letter and made the messages more clear.

Reply:
Fixed in https://github.com/openshift/cluster-dns-operator/pull/321/commits/d52dedae2997a56997be10a907ea34cf679c7132